### PR TITLE
Clarifying how far back lastsignindatetime goes

### PR DESCRIPTION
**Pull request by besiler**

### CODE REVIEW

**Review by FaithOmbongi — COMMENTED** [review]
Thank you @besiler for raising this PR. 

A point of confusion though: doesn't the note contradict the messaging we give about sign-in logs being governed by [Azure AD retention policies](https://docs.microsoft.com/en-us/azure/active-directory/reports-monitoring/reference-reports-data-retention#how-long-does-azure-ad-store-the-data)?

**Comment by besiler** [comment]
@FaithOmbongi we think of signinactivity as being a property of the user object rather than activity data, since it's a stable piece of info that is only updated periodically rather than a report of all activities that happened in the last 30 days. 

However, if you're raising this point we'll definitely have customers who make the same point. What if we added this disclaimer in the description signinactivity.md: "signinactivity is a resource that describes a property of the user object in your tenant. Azure AD stores signinactivity for your users for as long as the user objects exist."?

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Thank you @besiler for raising this PR. Looks good but for 2 gaps:
- Please document the 2 new members in tokenIssuerType enum
- Document the changelog

I also support your suggestion to enhance the resource description. Go for it.

File: api-reference/beta/resources/signinactivity.md [hunk 1]
```diff
@@ -19,9 +19,9 @@ Provides the last interactive or non-interactive sign-in time for a specific [us
...
+|lastNonInteractiveSignInDateTime|DateTimeOffset|The last non-interactive sign-in date for a specific user. You can use this field to calculate the last time a client signed in to the directory on behalf of a user. Because some users may use clients to access tenant resources rather than signing into your tenant directly, you can use the non-interactive sign-in date to along with lastSignInDateTime to identify inactive users. The timestamp represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is: `'2014-01-01T00:00:00Z'`. This property includes interactive sign-ins going back to May 2021; sign-ins from before that time will not be reflected in the property. For more information about using the value of this property, see [Manage inactive user accounts in Azure AD](/azure/active-directory/reports-monitoring/howto-manage-inactive-user-accounts).|
```
[the remark this answers is not in the document]
Comment (besiler) [reply api-reference/beta/resources/signinactivity.md:24]: Yep, that should be non-interactive

**Review by FaithOmbongi — APPROVED** [review]

**Comment by besiler** [comment]
Marking ready to merge since @FaithOmbongi approved

**Pull request merged**